androidApp/src/main/java/co/nimblehq/avishek/phong/kmmic/android/ui/common/HorizontalPagerIndicator.kt

```diff
@@ -12,18 +12,16 @@ import androidx.compose.ui.Modifier
 import androidx.compose.ui.graphics.Color
 import androidx.compose.ui.graphics.Shape
 import androidx.compose.ui.platform.LocalDensity
-import androidx.compose.ui.tooling.preview.Preview
 import androidx.compose.ui.unit.*
-import co.nimblehq.avishek.phong.kmmic.android.ui.theme.ApplicationTheme
 import kotlin.math.absoluteValue
 import kotlin.math.sign
 
 @OptIn(ExperimentalFoundationApi::class)
 @Composable
 fun HorizontalPagerIndicator(
     pagerState: PagerState,
-    pageCount: Int,
     modifier: Modifier = Modifier,
+    pageCount: Int,
```

Comment:
Why moved? 😆 It was right the first time. Please move it above modifier again. 🙏

Reply:
Rebase error. 😢